addons/performance/stringtable.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="TAC">
+    <Package name="Performance">
+        <Key ID="STR_TAC_Performance_EnabledRemoteSensors">
+            <English>Enable Remote Sensors on Clients</English>
+        </Key>
+        <Key ID="STR_TAC_Performance_EnabledRemoteSensorsDesc">
+            <English>This command will halt raycasting calculations (on the local machine only) for all groups which don't contain any local entities. Units, that are not in a group with at least one local member, will not check visibility of other units. This will cause, that remote units will not have updated knowsAbout and it will save some CPU time. If a group contains a single local entity then calculations will still be performed for the entire group.</English>
```

Comment:
```suggestion
            <English>This command will halt raycasting calculations (on the local machine only) for all groups which don't contain any local entities. Units, that are not in a group with at least one local member, will not check visibility of other units. This causes remote units not to have updated knowsAbout and it will save some CPU time. If a group contains a single local entity then calculations will still be performed for the entire group.</English>
```